src/test/test_ep_cred_common.h

```diff
@@ -148,12 +166,13 @@ test_parse_args(int argc, char **argv)
 		{"is_service",	no_argument,		0, 's'},
 		{"credits",	required_argument,	0, 'c'},
 		{"burst",	required_argument,	0, 'b'},
+		{"front_queue",	no_argument,		0, 'f'},
```

Comment:
minor...why is one QUEUE_FRONT and the other front_queue?

Reply:
missed during renaming change. will fix